Mathlib/Algebra/GroupPower/IterateHom.lean

```diff
@@ -95,20 +94,6 @@ theorem AddMonoid.End.coe_pow {A} [AddMonoid A] (f : AddMonoid.End A) (n : ℕ)
   hom_coe_pow _ rfl (fun _ _ => rfl) _ _
 #align add_monoid.End.coe_pow AddMonoid.End.coe_pow
 
-namespace RingHom
-
-section Semiring
-
-variable {R : Type*} [Semiring R] (f : R →+* R) (n : ℕ) (x y : R)
-
-theorem coe_pow (n : ℕ) : ⇑(f ^ n) = f^[n] :=
```

Comment:
According to an automated diff, this seems to be the only lemma that disappeared in this PR.

Reply:
Restored.